### PR TITLE
chore(deprecation): Remove Docker CIS from UI

### PR DESCRIPTION
**Pull request by connorgorman**
## Description

Removes Docker CIS from the FE after already removing it from the backend

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### Here I tell how I validated my change

UI e2e

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: ui/apps/platform/src/messages/standards.js [hunk 1]
```diff
@@ -6,8 +6,6 @@ export const standardLabels = {
     [standardTypes.NIST_SP_800_53_Rev_4]: 'NIST SP 800-53',
     [standardTypes.HIPAA_164]: 'HIPAA 164',
     [standardTypes.CIS_Kubernetes_v1_5]: 'CIS Kubernetes v1.5',
-    [standardTypes.CIS_Docker_v1_1_0]: 'CIS Docker v1.1.0',
```
Comment (vjwilson) [inline ui/apps/platform/src/messages/standards.js:9]: Hmm, looks like this is an artifact of an earlier removal.

**Review by vjwilson — APPROVED** [review]

File: ui/apps/platform/src/messages/standards.js (hunk 1 above)
Comment (connorgorman) [reply ui/apps/platform/src/messages/standards.js:9]: Yup, this is from previous cis docker version

**Pull request merged**